.bazelci/examples.yml

```diff
@@ -1,45 +1,23 @@
 ---
 tasks:
-  simple-linux:
```

Comment:
There is a lot of noise in this file because I sorted the entries when I added the java-export and kt_jvm_export examples.